pages/index.vue

```diff
@@ -15,17 +15,20 @@
           <Button>GitHub</Button>
         </a>
       </div>
+      <ListButton></ListButton>
```

Comment:
```suggestion
      <ListButton />
```

特に子要素に何も含まれないのであればこんな感じの書き方です。